src/components/BaseSelect.vue

```diff
@@ -27,6 +27,15 @@
             ></div>
           </div>
         </div>
+        <div class="mt-8">
+          <p class="text-left">Select which shade step you want the base color to apply at (default 500)</p>
+          <div class="flex mt-3">
+            <div v-for="stop in stops" :key="'stop-' + stop">
+              <input type="radio" v-model="baseShadeStop" name="baseShadeStop" :value="stop" class="ml-2" />
+              <label for="baseShadeStop" class="pl-2">{{ stop * 100 }}</label>
+            </div>
```

Comment:
I copied the `stops` from [this component](https://github.com/anheric/tailwindshades/blob/master/src/components/Shades.vue#L275) and then in the display multiplied them by 100, so `.5` in the data layer shows up as `50` in the UI